data_processing/transform.py

```diff
@@ -743,6 +744,105 @@ def load_category_and_sub_category():
                         convert_to_url_string(p[1]+p[2]), "category"])
         conn.commit()
 
+def ingest_additional_programs():
```

Comment:
nitpick here, lets name this to load_additional_programs for this function. Only reason is that it's more consistent with the the other functions in this file.